HealthTrack/app/src/androidTest/java/com/example/healthtrack/ExampleInstrumentedTest.java

```diff
@@ -17,6 +29,27 @@
  */
 @RunWith(AndroidJUnit4.class)
 public class ExampleInstrumentedTest {
+
+    @Rule
+    public ActivityTestRule<ExerciseActivity> activityTestRule = new ActivityTestRule<>(ExerciseActivity.class);
+
+    @Test
+    public void addToClickCounter() {
+        onView(withId(R.id.text_counter))
+                .perform(clearText())
+                .perform(typeText("10"));
+
+        for (int x = 0; x < 11; x++) {
+            onView(withId(R.id.count_button))
+                    .perform(click());
+        }
+        onView(allOf(withId(R.id.text_counter), withText("10")))

```

Comment:
When this test is run, it actually runs up to 11 clicks. This test fails because it's looking for 10, but actually hits 11.